### DISCUSSION TITLE
How to read javascript errors or the console log?

### DISCUSSION THREAD

**Discussion by uwe999** [vgalin/html2image#128] [General]
Love the tool!!!
One potential issue for my use case is that if the JavaScript included in my html throws an error the output doesn't render correctly.
Is there a way to detect JavaScript errors or read the browser's console log to detect such issues?
I see there is a new "disable_logging" logging feature but I didn't find any references about logs

**Comment by vgalin** [comment]
Hello, is it for a one-time debugging session ? If that's the case, you can take a look in the temporary directory of the package (`%TEMP%/html2image` on Windows). You'll find in this directory all the HTML files generated by the package and you'll be able to open them in your browser and see what's going on in the console.

**Reply by uwe999** [reply]
thanks, no, it would be to catch any errors when creating hundreds of screenshots in a batch.
I insert the images in videos so it would be great to check if there is a problem with the images first.
